source/MaterialXTest/Definition.cpp

```diff
@@ -0,0 +1,52 @@
+//
```

Comment:
Added in a new Definition test file as I didn't see any other good logical place to put this.